auth0/resource_auth0_branding.go

```diff
@@ -112,12 +111,10 @@ func readBranding(d *schema.ResourceData, m interface{}) error {
 	}
 
 	if t.Flags.EnableCustomDomainInEmails != nil && *t.Flags.EnableCustomDomainInEmails {
-		ul, err := api.Branding.UniversalLogin()
-		if err != nil {
```

Comment:
in the case of `t.Flags.EnableCustomDomainInEmails == true` and UniversalLogin is not set, the response is `404 not found` and branding cannot be updated.